src/tests.rs

```diff
@@ -527,6 +529,26 @@ mod system_macro {
                         &(Length::new::<meter>((*l).clone())
                             % Length::new::<meter>((*r).clone())).get(meter)))
             }
+
+            #[cfg(feature = "serde")]
+            #[allow(trivial_casts)]
+            fn serde_serialize(v: A<V>) -> bool {
+                let m = Length::new::<meter>((*v).clone());
+                let json_q = serde_json::to_string(&m).expect("Must be able to serialize Quantity");
+                let json_f = serde_json::to_string(&*v).expect("Must be able to serialize num");
+
+                json_f == json_q
+            }
+
+            #[cfg(feature = "serde")]
+            #[allow(trivial_casts)]
+            fn serde_deserialize(v: A<V>) -> bool {
+                let json_f = serde_json::to_string(&*v).expect("Must be able to deserialize num");
```

Comment:
I believe the `expect(...)` should say serialize and not deserialize.

Reply:
Right you are!